apps/search-poc/src/utils/__tests__/index.test.ts

```diff
@@ -1,140 +1,150 @@
-import { getLabel, parseNexusUrl, camelCaseToLabelString } from '../index';
+import {
+  getLabel,
+  parseNexusUrl,
+  camelCaseToLabelString,
+  camelToKebab
+} from "../index";
 
-describe('utils', () => {
-  describe('camelCaseToLabelString', () => {
-    it('should convert camel cased string to capitalized words', () => {
-      expect(camelCaseToLabelString('camelCase')).toEqual('Camel Case');
+describe("utils", () => {
```

Comment:
Why did we go from single quotes to double quotes? 🤔

Reply:
My prettier had a stroke! 🙈